jmix-flowui/flowui/src/main/java/io/jmix/flowui/facet/settings/component/binder/AbstractGridSettingsBinder.java

```diff
@@ -86,6 +88,7 @@ public void applyDataLoadingSettings(V component, S settings) {
         }
 
         List sortOrder = settings.getSortOrder().stream()
+                .filter(o -> !Objects.isNull(o.getKey()))
```

Comment:
if `SortOrder#getKey` can be `null` then it makes sense to add `@Nullable`

Reply:
solved by nullable annotation